packages/e2e/tests/connectButton/universal.e2e.ts

```
@@ -1,9 +1,20 @@
 import { type IInjectedProviderNames } from '@onekeyfe/cross-inpage-provider-types';
+import { expect, test } from '@playwright/test';
 import path from 'node:path';
 import { Locator } from 'playwright/test';
-import { sitesConfig } from '../config';
-import { createWalletId } from '../utils';
-import { expect, test } from './fixtures';
+
+import * as fs from 'fs';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)